pandas/core/indexes/datetimes.py

```diff
@@ -1743,7 +1743,33 @@ def freq(self, value):
     is_year_start = _field_accessor(
         'is_year_start',
         'is_year_start',
-        "Logical indicating if first day of year (defined by frequency)")
+        """
+        Return a boolean indicating whether the date is the first day of the
+        year.
+
+        Returns
+        -------
+        is_year_start : Series of boolean.
+
+        See Also
+        --------
+        is_year_end : Return a boolean indicating whether the date is the
+            last day of the year.
+
+        Examples
+        --------
+        >>> dates = pd.Series(pd.date_range("2017-12-30", periods=3))
+        >>> dates
+        0   2017-12-30
+        1   2017-12-31
+        2   2018-01-01
+        dtype: datetime64[ns]
+        >>> dates.dt.is_year_start
+        0    False
+        1    False
+        2    True
+        dtype: bool
```

Comment:
Second example showing DatetimeIndex.is_year_start.

Reply:
Can you then also write something like "This attribute is both available on a DatetimeIndex as a Series with datetime values" or for the Series example: "On a Series with datetime values, the attribute can be accessed with `.dt`"